utils.h

```diff
@@ -143,6 +143,13 @@ gboolean janus_flags_is_set(janus_flags *flags, gsize flag);
  * @note A failure may indicate that creating any of the subdirectories failed: some may still have been created */
 int janus_mkdir(const char *dir, mode_t mode);
 
+/*! \brief Helper to convert \c path relative to \c base_dir to absolute path.
+ *  If \c path already represents absolute path then just g_strdup it.
+ * @param[in] base_dir Path wich will be prepended to \c path if it's relative
+ * @param[in] path
```

Comment:
It looks like `path` is not documented?

Reply:
I've just decided `path` meaning is clear enough from other parts of description. But I can add something there if it's required.

Reply:
We try to document everything, so that would help. There's also a typo (wich) in the line above, by the way (sorry for missing it before!)

Reply:
Np, I'll extend it.

P.S. English is not my native language so I surprised there is only one typo there :smile: